tensorboard/plugins/interactive_inference/witwidget/notebook/jupyter/js/package.json

```diff
@@ -16,8 +14,8 @@
     "jupyterlab-extension"
   ],
   "files": [
-    "lib/**/*.js",
-    "dist/*.js"
+    "lib/*",
```

Comment:
totally up to you, but longterm might be nice to have both of them use dist? (Though there may be valid reasons to have them use diff sources/builds)

Reply:
done, removed lib

Reply:
was actually mistaken about this - labextensions fail to install unless the npm package contains lib/labplugin.js - added it back in